DeepSpeech.ipynb

```diff
@@ -1337,6 +1351,55 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Finally, we restore the trained variables into a simpler graph that we can export for serving."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "with tf.device('/cpu:0'):\n",
+    "    model_exporter = exporter.Exporter(saver)\n",
+    "    \n",
+    "    # Run inference\n",
+    "    input = tf.placeholder(tf.float32, [None, None, n_input + 2*n_input*n_context])\n",
```

Comment:
Could you choose another variable name, there's a builtin function named `input`. From `input`'s help page
```
Help on built-in function input in module __builtin__:

input(...)
    input([prompt]) -> value
    
    Equivalent to eval(raw_input(prompt)).
```